### ISSUE TITLE
Latex error converting to DVI.

### ISSUE THREAD

**Issue by spicethings** [3b1b/manim#768]
Howdy hey. I'm trying to get manim working, and SquareToCircle is running fine. I'm trying to run OpeningManimExample, however, I get this error.

`Traceback (most recent call last):
  File "C:\Users\Spice\Downloads\Manim Setup\manim-0.1.10\manimlib\extract_scene.py", line 150, in main
    scene = SceneClass(**scene_kwargs)
  File "C:\Users\Spice\Downloads\Manim Setup\manim-0.1.10\manimlib\scene\scene.py", line 52, in __init__
    self.construct()
  File "example_scenes.py", line 20, in construct
    title = TextMobject("This is some \\LaTeX")
  File "C:\Users\Spice\Downloads\Manim Setup\manim-0.1.10\manimlib\mobject\svg\tex_mobject.py", line 144
    self, self.arg_separator.join(tex_strings), **kwargs
  File "C:\Users\Spice\Downloads\Manim Setup\manim-0.1.10\manimlib\mobject\svg\tex_mobject.py", line 44,
    self.template_tex_file_body
  File "C:\Users\Spice\Downloads\Manim Setup\manim-0.1.10\manimlib\utils\tex_file_writing.py", line 19,
le
    dvi_file = tex_to_dvi(tex_file)
  File "C:\Users\Spice\Downloads\Manim Setup\manim-0.1.10\manimlib\utils\tex_file_writing.py", line 67,
    "See log output above or the log file: %s" % log_file)
Exception: Latex error converting to dvi. See log output above or the log file: Tex\081b5f87a6472151.log`

I'm running Manim-0.1.10 on Windows 10. Help would be appreciated, thanks.

**Comment by maxlukian** [comment]
I am having a similar issue. Does your tex_file_writing.py look like this?

<img width="662" alt="Screen Shot 2019-10-19 at 3 50 50 PM" src="https://user-images.githubusercontent.com/56772649/67151375-6a5b6680-f293-11e9-999e-5ab4bd75210e.png">

**Comment by Tridu33** [comment]
Here come the same problem.
If I use TextMobject()，it throw out this exception.
I use windows10,miktex，i check my iftex package in miktex，it seems I have installed.
But manim said package iftex  couldn't be installed

**Comment by Tridu33** [comment]
I had solved my problem.
find your file,open  console and write:
latex 8d5v7c*.tex
to create 8d5v7c*.dvi  files，and them write 
dvisvgm 8d5v7c*.dvi
to create 8d5v7c*.svg which can be loaded in manim.

Usually,if there wrongs happened,please check your 8d5v7c*.tex file,although you can get 8d5v7c*.pdf by write:
pdflatex 8d5v7c*.tex
or you write :
xelatex 8d5v7c*.tex
there maybe bug happened when you use dvisvgm command.

To sum up,
1. check your 8d5v7c*.tex file if it is right?
2.update your latex packages,take me for example,i reinstall my Ctex pavkages and run script: upmap --force 
and it work.

so ,it is problem about latex but not manim

**Comment by edjorqs** [comment]
@Tridu33  What is 8d5v7c*.tex in your system?

**Comment by arpon113** [comment]
> I had solved my problem.
> find your file,open console and write:
> latex 8d5v7c*.tex
> to create 8d5v7c*.dvi files，and them write
> dvisvgm 8d5v7c*.dvi
> to create 8d5v7c*.svg which can be loaded in manim.
> 
> Usually,if there wrongs happened,please check your 8d5v7c*.tex file,although you can get 8d5v7c*.pdf by write:
> pdflatex 8d5v7c*.tex
> or you write :
> xelatex 8d5v7c*.tex
> there maybe bug happened when you use dvisvgm command.
> 
> To sum up,
> 
> 1. check your 8d5v7c*.tex file if it is right?
>    2.update your latex packages,take me for example,i reinstall my Ctex pavkages and run script: upmap --force
>    and it work.
> 
> so ,it is problem about latex but not manim

If you kindly explain the action it would be very helpful. I am having a hard time to catch up. I havenot used latex and dont have much knowledge about latex codes. Thanks
